### PR TITLE
docs: update default network_params.json file

### PR DESCRIPTION
**Pull request by leeederek**
- reduce capella fork epoch to 0 from 2
- indent mev_params for easier readability
- add "mev_flood_seconds_per_bundle": 15,
- add "launch_custom_flood": false

### CODE REVIEW

File: network_params.json [hunk 1]
```diff
@@ -33,12 +33,14 @@
...
     "mev_type": "None",
     "mev_params": {
-		"mev_relay_image": "flashbots/mev-boost-relay",
-		"mev_relay_api_extra_args": [],
-		"mev_relay_housekeeper_extra_args": [],
-		"mev_relay_website_extra_args": [],
-		"mev_builder_extra_args": [],
-		"mev_flood_image": "flashbots/mev-flood",
-		"mev_flood_extra_args": []
+		    "mev_relay_image": "flashbots/mev-boost-relay",
```
Comment (h4ck3rk3y) [inline network_params.json:36]: indentation needs imprvoement

File: network_params.json [hunk 2]
```diff
@@ -1,44 +1,46 @@
...
+		"deneb_fork_epoch": 4
+	},
+	"launch_additional_services": true,
+	"wait_for_finalization": false,
+	"wait_for_verifications": false,
+	"verifications_epoch_limit": 5,
+	"global_client_log_level": "info",
+	"snooper_enabled": false,
+	"parallel_keystore_generation": false,
+	"mev_type": "none",
```
Comment (h4ck3rk3y) [inline network_params.json:34]: Capital "None" please, or `null` is even nicer

File: network_params.json [hunk 3]
```diff
@@ -1,44 +1,46 @@
...
+		}
+	],
+	"network_params": {
+		"network_id": "3151908",
+		"deposit_contract_address": "0x4242424242424242424242424242424242424242",
+		"seconds_per_slot": 12,
+		"slots_per_epoch": 32,
+		"num_validator_keys_per_node": 64,
+		"preregistered_validator_keys_mnemonic": "giant issue aisle success illegal bike spike question tent bar rely arctic volcano long crawl hungry vocal artwork sniff fantasy very lucky have athlete",
+		"capella_fork_epoch": 1,
```
Comment (h4ck3rk3y) [inline network_params.json:24]: this should be 0

File: network_params.json [hunk 4]
```diff
@@ -1,44 +1,46 @@
...
+		"deneb_fork_epoch": 4
```
Comment (h4ck3rk3y) [inline network_params.json:25]: this should be 500

**Review by h4ck3rk3y — CHANGES_REQUESTED** [review]

File: network_params.json (hunk 3 above)
Comment (leeederek) [inline network_params.json:24]:
```suggestion
		"capella_fork_epoch": 0,
```

File: network_params.json (hunk 4 above)
Comment (leeederek) [inline network_params.json:25]:
```suggestion
		"deneb_fork_epoch": 500
```

File: network_params.json (hunk 2 above)
Comment (leeederek) [inline network_params.json:34]:
```suggestion
	"mev_type": "None",
```

Comment (h4ck3rk3y) [reply network_params.json]: make it null

**Review by h4ck3rk3y — APPROVED** [review]
@leeederek looks good just testing, 1min

File: network_params.json [hunk 5]
```diff
@@ -1,44 +1,46 @@
...
-    "launch_additional_services": true,
-    "wait_for_finalization": false,
-    "wait_for_verifications": false,
-    "verifications_epoch_limit": 5,
-    "global_client_log_level": "info",
-    "snooper_enabled": false,
-    "parallel_keystore_generation": false,
-    "mev_type": "None",
-    "mev_params": {
+	"participants": [
```
Comment (h4ck3rk3y) [inline network_params.json:2]: prevent merge

File: network_params.json [hunk 6]
```diff
@@ -1,44 +1,46 @@
...
+		{
```
Comment (h4ck3rk3y) [inline network_params.json:3]: block

Comment (h4ck3rk3y) [reply network_params.json:3]: for @barnabasbusa @parithosh to resolve for merge

**Review by parithosh — APPROVED** [review]

**Review by barnabasbusa — APPROVED** [review]

**Pull request merged**